ws-service/server.ts

```diff
@@ -44,25 +44,55 @@ const addressRouteConnection = (socket: Socket): void => {
   console.log('  total:', countA)
 }
 
+const resolveOpReturn = (opr: any) => {
+  try {
+    if (!(typeof opr === 'string' && opr !== '')) {
+      throw new Error("Error parsing Opreturn options")
+    }
+    return JSON.parse(opr)
+  } catch (e) {
+    return undefined
```

Comment:
we were using `null` before, which I think is slightly better because there is less ambiguity with `undefined`s that happen simply because the expected data was not there.